Libraries/Components/View/View.js

```diff
@@ -202,6 +203,14 @@ const View = React.createClass({
     onMoveShouldSetResponderCapture: PropTypes.func,
 
     /**
+     * This defines how far a touch event can start away from the view.
+     * The touch area never extends past the parent view bounds and the Z-index
+     * of sibling views always takes precedence if a touch hits two overlapping
+     * views.
```

Comment:
Can you add a short example here showing how I would extend the area by n pixels in each direction? What's a reasonable number?